### PR TITLE
test: change test image

### PR DESCRIPTION
**Pull request by rudyfly**
### Ⅰ. Describe what this PR did
change test image to `busyboxImage`

### Ⅱ. Does this pull request fix one issue?

NONE

### Ⅲ. Why don't you add test cases (unit test/integration test)? (你真的觉得不需要加测试吗？)
just modify test case.

### Ⅳ. Describe how to verify it
test case: `TestCommitHardLink`, `TestLogsOpt`, `TestLogsWithDetails` pass

### Ⅴ. Special notes for reviews

Signed-off-by: Rudy Zhang <rudyflyzhang@gmail.com>

### CODE REVIEW

File: test/cli_container_commit_test.go [hunk 1]
```diff
@@ -64,7 +64,7 @@ func (suite *PouchCommitSuite) TestCommitHardLink(c *check.C) {
 	cname := "TestCommitHardLink"
 	image := "foo:hardlink"
 
-	ret := command.PouchRun("run", "-t", "--name", cname, "busybox", "sh", "-c", "touch file1 && ln file1 file2 && ls -di file1 file2")
+	ret := command.PouchRun("run", "-t", "--name", cname, busyboxImage, "sh", "-c", "touch file1 && ln file1 file2 && ls -di file1 file2")
```
Comment (HusterWan) [inline test/cli_container_commit_test.go]: convert one line to multi line?

Comment (rudyfly) [reply test/cli_container_commit_test.go]: modified

**Comment by rudyfly** [comment]
PTAL @HusterWan

**Comment by HusterWan** [comment]
LGTM

**Pull request merged**